src/index.ts

```diff
@@ -1676,6 +1680,65 @@ class Spanner extends GrpcService {
   static pgJsonb(value): PGJsonb {
     return new codec.PGJsonb(value);
   }
+
+  /**
+   * @typedef IProtoMessageParams
+   * @property {object} value Proto Message Value. Serialized Buffer or message object.
+   * @property {string} fullName Proto message Full Qualified Path Name.
```

Comment:
Does the upper casing on the comments refer to something special like a type name? E.g. "Serialized Buffer", "Full Qualified Path Name". If not, then just writing a sentence like "Fully-qualified path name of proto message." is preferable.